TFT/src/User/Menu/PrintingMenu.c

```diff
@@ -236,7 +236,7 @@ static inline void reDrawPrintingValue(uint8_t icon_pos, uint8_t draw_type)
       case ICON_POS_Z:
         if (layerDisplayType == SHOW_LAYER_BOTH)
         {
-          sprintf(tempstr, "%6.2fmm", (infoFile.source >= BOARD_SD) ? coordinateGetAxisActual(Z_AXIS) : coordinateGetAxisTarget(Z_AXIS));
+          sprintf(tempstr, "%3.2fmm", (infoFile.source >= BOARD_SD) ? coordinateGetAxisActual(Z_AXIS) : coordinateGetAxisTarget(Z_AXIS));
```

Comment:
It's me who put %6 there. "%6.2f" means **_at least_** 6 characters before the decimal point. I did it so the displayed text to be centered in the display area, it is not a mistake. 
This is for the lower row of the layer display area. 
For the upper row I put %3 because of the different behaviour of the used method to display text. A left aligned method  is used there. 3 digits before the decimal point is enough, I do not think there's a 3D printer that can print above 999.99mm

Reply:
Yes I know but the size of tempstr is 10 chars. 6.2mm + '\0'gives 12 :-) so something should be removed (e.g. mm). No problem to revert my change but consider my suggestion

Reply:
For some reason unknown to me it is working just fine the way it is.